lib/QC_TreeMap.qpp

```diff
@@ -88,13 +88,31 @@ nothing TreeMap::put(string path, any value) {
    tm->put(path, value, xsink);
 }
 
-//! Retrieves a value from the TreeMap.
+//! Retrieves a value from the TreeMap and returns optionally unmatched path suffix
 /** Looks for an entry whose key is the longest prefix of \c path.
     @param path the path to lookup
+    @param unmatched path suffix, reference to a *string compatible type is needed, if \c path not found then returns NOTHING,  @since %Qore 0.8.13
```

Comment:
also the `@since` lines should be included as separate lines - like this
```
@param unmatched an output variable returning the unmatched path suffix, reference to a ref @string_or_nothing_type "*string" compatible type is needed, if \c path is not matched then @ref nothing is stored here

@since %Qore 0.8.13
```